projects/opendr_ws/src/perception/scripts/point_cloud_dataset.py

```diff
@@ -36,6 +38,7 @@ def __init__(
         self.dataset = dataset
         # Initialize OpenDR ROSBridge object
         self.bridge = ROSBridge()
+        self.delay = 1.0 / data_fps
 
         if output_point_cloud_topic is not None:
```

Comment:
I actually missed this in the original review, this if can be removed as there's no reason for a user to disable this node's single output.

Reply:
Updated